buildSrc/src/main/kotlin/addMavenCentralRepository.kt

```diff
@@ -0,0 +1,50 @@
+import com.google.common.base.CaseFormat
+import org.gradle.api.Project
+import org.gradle.api.publish.PublishingExtension
+import org.gradle.api.publish.maven.MavenPublication
+import org.gradle.api.publish.maven.tasks.PublishToMavenRepository
+import org.gradle.kotlin.dsl.withType
+import java.net.URI
+
+fun PublishingExtension.addMavenCentralRepository(project: Project, publication: MavenPublication) {
+    val sonatypeApiUser = project.providers.gradleProperty("sonatypeApiUser")
+        .forUseAtConfigurationTime()
+        .orElse(project.providers.environmentVariable("SONATYPE_API_USER").forUseAtConfigurationTime())
+    val sonatypeApiKey = project.providers.gradleProperty("sonatypeApiKey")
+        .forUseAtConfigurationTime()
+        .orElse(project.providers.environmentVariable("SONATYPE_API_KEY").forUseAtConfigurationTime())
+    if (!sonatypeApiUser.isPresent || !sonatypeApiKey.isPresent) {
+        project.logger.info("Sonatype API key not defined, skipping configuration of Maven Central publishing repository")
+        return
+    }
+
+    val camelCaseArtifactId = CaseFormat.LOWER_HYPHEN.to(CaseFormat.UPPER_CAMEL, publication.artifactId)
+
+    val mavenRepo = repositories.maven {
+        name = "sonatype$camelCaseArtifactId"
+        url = URI("https://oss.sonatype.org/service/local/staging/deploy/maven2")
```

Comment:
Single instead of dynamic URI. 

Note that Maven Central doesn't have the `repositories` classification as what Bintray had. But `pbandk` already has it's artifact name in the `groupid` so developers can still easily discover all `pbandk` artifacts by searching for the `groupid`.

Similar to retrofit: https://search.maven.org/search?q=g:com.squareup.retrofit2